src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -1075,9 +1078,12 @@ class BeginStatementSegment(BaseSegment):
             "ERROR",
             "THEN",
             Indent,
-            Sequence(
-                Ref("StatementSegment"),
-                Ref("DelimiterGrammar"),
+            AnyNumberOf(
```

Comment:
@alanmcruickshank I realised after my last PR that my change was only going to be limited to a single statement, and in some of our use cases there will be multiple things happening in `BEGIN` or `EXCEPTION`.

I've used `AnyNumberOf` which worked to allow multiple statements in the `BEGIN` block, but for some reason the same thing is not applying to the `EXCEPTION` and unable to generate the YAML.

I'm wondering if you can see something obviously wrong here?

Reply:
I think I figured something out 😄